### ISSUE TITLE
Can't read properties of null error upon opening the website

### ISSUE THREAD

**Issue by aneesafatima** [sugarlabs/www#564]
When I am opening the website, I get this error. 

![Screenshot (478)](https://github.com/user-attachments/assets/41555b13-e9c6-4bfb-a71e-ee07d9d829d7)

**Comment by quozl** [comment]
Please use text so it can be searched.  Firefox gives this in console;

```
Uncaught TypeError: document.querySelector(...) is null
    <anonymous> https://www.sugarlabs.org/js/main.js:15
    jQuery 4
```

**Comment by quozl** [comment]
You've not searched in time dimension.  Please do use git to examine history closely.  Using emacs ang magit, listing commits limited to changes containing `logo1`, elements with this class name were removed three years ago by 499babf, and a month ago by 64b1f9f and 7088c6d.  It seems likely that @therealharshit introduced this fault, and that testing of pull request https://github.com/sugarlabs/www/pull/503 by participants did not include console messages.

**Comment by aneesafatima** [comment]
Since those elements no longer exist, I think that removing code involving them will not damage the website and will also fix the bug

**Comment by quozl** [comment]
Possibly you are right.  However, could it be that removing the elements was the mistake?  What was the code for?  If it was to randomly replace colours in the style of the Sugar learning environment (one of our primary software projects), have we lost that feature?

**Comment by yashsaraswat2004** [comment]
Can I work on this issue?

**Comment by quozl** [comment]
Welcome.  Sorry, I don't understand.  Why can't you fix it without asking?  You are asking for the right to be the only one to fix it?  We can't offer that right.  Anyone could fix it.  Might be you, might be someone else, might be a group.

If you'd like to work on this issue, I think the next step is to answer my questions in the comment above; i.e. find out why this has become a problem, and what has become of the random colour feature we used to have.

**Comment by therealharshit** [comment]
> Possibly you are right. However, could it be that removing the elements was the mistake? What was the code for? If it was to randomly replace colours in the style of the Sugar learning environment (one of our primary software projects), have we lost that feature?

@quozl I have restored the color changing logo feature which also fixes this issue, please review it in my PR #567 .
